components/Tabs/tests/unit/MaterialTabsTests.m

```diff
@@ -96,6 +99,30 @@ - (void)testItemsUpdateIsIdempotent {
   XCTAssertEqual(_tabBar.selectedItem, _itemB);
 }
 
+- (void)testInkStyleForBottom {
```

Comment:
Tip: Bottom Tabs style should not receive feature improvements any more. We now have Bottom Navigation which is a dedicated component to handle this use case.